stm32data/__main__.py

```diff
@@ -438,7 +438,11 @@ def cleanup_pin_name(pin_name):
 
 
 def parse_signal_name(signal_name):
-    parts = signal_name.split('_', 1)
+    if signal_name.startswith('USB_OTG_FS') or signal_name.startswith('USB_OTG_HS'):
```

Comment:
I couldn't do a clever solution like in `parse_chips()`, because peripheral names are not available at this point. Would it make sense to change `parse_chips` function into the same `if` statements for faster processing? TBH, I did not notice any slowdown due to array lookup.

Reply:
If it's not slower then it's fine :)